### DISCUSSION TITLE
Map markers are moving when zoom-in/zoom-out

### DISCUSSION THREAD

**Discussion by lampn9397** [rnmapbox/maps#3203] [Q&A] [answered]
Hi everyone,

I'm facing the issue that the map markers are moving when zoom-in/zoom-out, but expected is the markers are staying on it original coordinate. Does anyone know how to manage it?

https://github.com/rnmapbox/maps/assets/22604768/015cdf65-bc90-41b7-a1bc-173990194973

**Comment by mfazekas** [comment, marked as answer]
You should configure Anchor of MarkerView if that what's you're using: https://rnmapbox.github.io/docs/components/MarkerView#anchor

**Reply by lampn9397** [reply]
Thanks @mfazekas! That's what I was looking for.
